### PR TITLE
fix(layers): Extra check to the doRefresh property

### PR DESCRIPTION
**Pull request by tombatossals**
Now tests pass again

### CODE REVIEW

**Comment by tombatossals** [comment]
@nmccready solved the merge problems in this PR, good seen

File: src/directives/layers.js [hunk 1]
```diff
@@ -81,9 +81,9 @@ angular.module("leaflet-directive").directive('layers', function ($log, $q, leaf
                             }
                             delete leafletLayers.baselayers[name];
 
-                            // if (newBaseLayers[name].doRefresh) {
-                            //     newBaseLayers[name].doRefresh = false;
-                            // }
+                            if (newBaseLayers[name] && newBaseLayers[name].doRefresh) {
```
Comment (nmccready) [inline src/directives/layers.js]: why is this necessary?

Comment (tombatossals) [inline src/directives/layers.js]: @nmccready, I was surprised about this like you, but it seems that the newBaseLayers[name] item can be undefined. The tests didn't pass if it's not present that line.

I was surprised that travis didn't catch this too. Do you know why this could have happened?

screenshot from 2015-04-25 15-28-07

**Comment by nmccready** [comment]
ok LGTM
